### PR TITLE
Improve the 3D View tool Composited Layers docs and remove the Layers tool page

### PR DESCRIPTION
**Pull request by captainbrosset**
The Layers tool is going away (it already redirected to the 3D view tool, but now it's being removed completely).

This PR updates the docs accordingly: it removes the Layers tool page entirely. I also took this opportunity to improve the Composited Layers view part of the 3D view tool docs.

**Rendered articles or article sections for review:**
* https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/3d-view/index?branch=pr-en-us-1994
* https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools?branch=pr-en-us-1994#layers-in-3d-view
* https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2020/11/devtools?branch=pr-en-us-1994#composited-layers-are-now-in-3d-view

**Redirection file entry to test:**
* https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/layers/layers-tool?branch=pr-en-us-1994
Passed; you end up at:
* https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/3d-view/?branch=pr-en-us-1994

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/about-tools.md [hunk 1]
```diff
@@ -49,7 +49,6 @@ Microsoft Edge DevTools includes the following tools.
 | **Inspect** tool | Use the **Inspect** tool to see information about an item within a rendered webpage.  When the **Inspect** tool is active, you _hover_ over items in the webpage, and DevTools adds an information overlay information and grid highlighting on the webpage. | [Analyze pages using the Inspect tool](css/inspect.md) |
 | **Issues** tool | The **Issues** tool automatically analyzes the current webpage, reports issues grouped by type, and provides documentation to help explain and resolve the issues. | [Find and fix problems using the Issues tool](issues/index.md) |
 | **JavaScript Profiler** tool | This tool has been replaced by the **Performance** tool and the **Memory** tool. | [JavaScript Profiler tool](javascript-profiler/javascript-profiler-tool.md) |
-| **Layers** tool | The **Layers** tool represents the webpage in a 3D space.  The **Layers** tool has essentially been replaced by the **3D View** tool. | [Layers tool](layers/layers-tool.md) |
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/about-tools.md:52]: The redirects file to update: edge-developer\.openpublishing.redirection.json

Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/about-tools.md:52]: I added a redirection rule to the json file and tested it on the preview site. It seems to work fine.

File: microsoft-edge/devtools-guide-chromium/layers/layers-tool.md [hunk 2]
```diff
@@ -1,23 +0,0 @@
...
-
-The **Layers** tool (now the **3D View** tool) represents the webpage in a 3D space, to help you improve the rendering performance of your webpage.  Use the **Layers** tool to analyze how many layers are created by your CSS code, how big they are, and how often they change.  The **Layers** tool shows the different layers that are created by the browser rendering engine, and then displays a composited image of the webpage in a 3D space that you can rotate.
-
-The **Layers** tool is now superseded by the **Composited Layers** tab of the **3D View** tool.  The following screenshot shows the **Layers** tool open in **Quick View** using **Focus Mode**, instead of in the Drawer:
-
-![The Layers tool now links to the 3D View tool.](images/layers-link-3d-view-tool.png)
-
-The **Layers** tool previously showed the following:
-
-![The Layers tool.](../media/layers-tool.png)
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/layers/layers-tool.md:23]: fulltext search repo for "layers-tool.png".  if no file uses this png, delete this png.

Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/layers/layers-tool.md:23]: Good point. No files use this, so I deleted it.

**Comment by captainbrosset** [comment]
This is no longer on hold. The Layers panel has now been removed, so I'll go ahead and merge this.

**Pull request merged**